src/test/java/org/jitsi/meet/test/base/AbstractBaseTest.java

```diff
@@ -177,6 +184,23 @@ private Properties mergeTestNGSuiteProperties(
         return participants.getAll();
     }
 
+    /**
+     * Return new {@link JitsiMeetUrl} instance which has only
+     * {@link JitsiMeetUrl#serverUrl} field initialized with the value from
+     * {@link #JITSI_MEET_URL_PROP} system property.
+     *
+     * @return a new instance of {@link JitsiMeetUrl}.
+     */
+    public JitsiMeetUrl getJitsiMeetUrl()
+    {
+        JitsiMeetUrl url = new JitsiMeetUrl();
+
+        url.setServerUrl(System.getProperty(JITSI_MEET_URL_PROP));
```

Comment:
can we use config instead of System.getProperty ? Eventually this could come from the factory which stores the config already.